kentikapi/client.go

```diff
@@ -31,6 +38,13 @@ type Client struct {
 	Alerting *resources.AlertingAPI
 
 	config Config
+
+	// cloudexport
```

Comment:
As we're at it, IMO such comments bring no value and should be removed. The same with _// synthetics_ below

Reply:
Done